app/controllers/v0/forms_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module V0
+  class FormsController < ApplicationController
+    include ActionView::Helpers::SanitizeHelper
+
+    skip_before_action :authenticate
+
+    def index
+      response = Forms::Client.new(query).get_all
+      render json: response.body,
+             status: response.status
+    end
+
+    def query_params
+      params.permit(:query)
+    end
+
+    def query
+      sanitize query_params['query']
+    end
```

Comment:
"query" and "params" gets used a lot here -- seems like it might be clearer with something like

`Forms::Client.new(search_term)` -- assuming its a search term. query is a pretty generic word